web/01-nginx-base/install.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+set -o xtrace
+
+# these installs are necessary to enable webdav class 2
+# https://github.com/arut/nginx-dav-ext-module
+apt-get -qq update
+apt-get -qq -y --no-install-recommends install \
+    nginx-extras \
+    libnginx-mod-http-dav-ext
+rm -rf /var/lib/apt/lists/*
+
+dpkg --list
```

Comment:
This just for debugging purposes?

Reply:
Yeah, as additional output during the build. It's somewhat redundant b/c we can always run this command against the image itself; but nice to be able to do it from CI w/o needing to pull the image if something goes wrong or the image isn't published.